view_all_set.php

```diff
@@ -359,6 +360,16 @@
 $f_relationship_type = gpc_get_int( FILTER_PROPERTY_RELATIONSHIP_TYPE, -1 );
 $f_relationship_bug = gpc_get_int( FILTER_PROPERTY_RELATIONSHIP_BUG, 0 );
 
+# TODO @cproensa
+# It is assumed that 'temporary' parameter means setting a new filter, so here type is set to 1
+# however, the temporary nature has to be carried through the pagination links, and the column sorting
+# Currently this is achieved by passing the filter token id (stored temporary filter)
+# BUT current_user_get_bug_filter() doesnt use that number at all, only if its present,
+# and it is a number, it gets the filter from token table, regardless of its id
+# Proposal is to change semantics:
+# * 'temporary' means it is stored/read from tokens table (no id, just current user)
+#   and carry this parameter through pagination and sorting
+# * caller, eg, my_view, should set type=1 on first call, to force creation of new temporary filter.
```

Comment:
For this comment i didnt account for actually using the token ids to allow multiple temporary filters per user, by actually using the passed id number